### PR TITLE
Stylistic changes

### PR DESCRIPTION
**Pull request by gracetian6**
website look:
https://drive.google.com/file/d/1GbjcWOeASgguyWwTuSw4dYA66BEVfFEC/view?usp=sharing

### CODE REVIEW

File: portfolio/src/main/webapp/data.js [hunk 1]
```diff
@@ -1,66 +1,24 @@
 // arrays store text that will be displayed on pages 
 window.moments = [
-    `I did math research on knots in high school, 
-    so my goal was to prove new patterns about these 
-    geometrical figures. So, I spent a lot of time 
-    drawing a lot of diagrams to test out new ideas.`,
+    `I did math research on knots in high school, so my goal was to prove new patterns about these geometrical figures. So, I spent a lot of time drawing a lot of diagrams to test out new ideas.`,
```
Comment (jpnickolas) [inline portfolio/src/main/webapp/data.js]: Please revert these. Your javascript should never have more than 80 characters on a line.

File: portfolio/src/main/webapp/moments.html [hunk 2]
```diff
@@ -9,11 +9,12 @@
...
     <div id="content">
-      <h1>Random Moments in my Life</h1>
-      <p>Click the button below to see a random picture of my life:</p>
+      <h1>A Moment in my Life</h1>
       <button onclick="randomizeImage()">New Picture</button>
-      <div id="random-image-container" class="image-thumbnails"></div>
-      <div id="random-caption-container"></div>
+      <div class="row">
+        <div id="random-caption-container" class="column"></div>
+        <div id="random-image-container" class="image-thumbnails" class="column"></div>
```
Comment (jpnickolas) [inline portfolio/src/main/webapp/moments.html]: Multiple classes can sit in the same attribute:
```
class="image-thumbnails column"
```

File: portfolio/src/main/webapp/quotes.html [hunk 3]
```diff
@@ -4,12 +4,12 @@
...
     <title>DEEP Quotes</title>
     <link rel="stylesheet" href="style.css">
+    <script src="data.js"></script>
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>Random Quotes that Resonate with me</h1>
-
+    </div>
```
Comment (jpnickolas) [inline portfolio/src/main/webapp/quotes.html]: Format empty html on one line like: `<div id="content"></div>`

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -16,14 +17,97 @@
...
+body{
+  height: calc(100vh - 8em);
+  padding: 4em;
+  color: rgba(255,255,255,.75);
+  Font-Family: 'Anonymous Pro', Monospace;
+  Font-Size: 19px;
+  background-color: #013220; 
+}
+
+.line-1{
```
Comment (jpnickolas) [inline portfolio/src/main/webapp/style.css]: line-1 is ambiguous. Can you give this class a more descriptive name?

**Review by jpnickolas — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -2,24 +2,20 @@
...
-      </p>
-
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Grace Tian.</h1>
+      <p>I'm a Noogler. An Ohioan. A Harvard sophomore.</p>
+      <p>I'm also a foodie. A kdrama fan.</p>
+      <p>I'm inspired by creative solutions and personal growth.</p>
+      <p> Learn more about my </p>
```
Comment (jessicaslaughter) [inline portfolio/src/main/webapp/index.html]: nit: remove extra spaces inside the paragraph tags

File: portfolio/src/main/webapp/data.js (hunk 1 above)
Comment (gracetian6) [reply portfolio/src/main/webapp/data.js]: When added line breaks previously then these line breaks would show up in the rendered page. I didn't know how to change it so I kept the line long. Is there a way to keep it multi-line without translating to line breaks on the webpage?

File: portfolio/src/main/webapp/style.css [hunk 6]
```diff
@@ -16,14 +17,97 @@
...
+    top: 50%;  
+    width: 24em;
+    margin: 0 auto;
+    text-align: center;
+    white-space: nowrap;
+    overflow: hidden;
+    transform: translateY(-50%);    
+}
+
+h1{
```
Comment (jessicaslaughter) [inline portfolio/src/main/webapp/style.css]: nit: there should always be a space after the last selector and the opening brace in a declaration block

some more guidance on that: https://google.github.io/styleguide/htmlcssguide.html#Declaration_Block_Separation

File: portfolio/src/main/webapp/data.js (hunk 1 above)
Comment (jpnickolas) [reply portfolio/src/main/webapp/data.js]:
```
const moments = [
  'some text that is'
  + 'going to be very long.',
  ....
];
```

File: portfolio/src/main/webapp/style.css [hunk 7]
```diff
@@ -16,14 +17,97 @@
...
+  Font-Family: 'Lora';
+  Font-Size: 52px;
```
Comment (jessicaslaughter) [inline portfolio/src/main/webapp/style.css]: nit: these should be lowercase

style guidance: https://google.github.io/styleguide/htmlcssguide.html#Capitalization

File: portfolio/src/main/webapp/style.css [hunk 8]
```diff
@@ -16,14 +17,97 @@
...
+a {
+  background:
+     linear-gradient(
+       to bottom, var(--mainColor) 0%,
+       var(--mainColor) 100%
+     );
+    background-position: 0 100%;
+    background-repeat: repeat-x;
+    background-size: 4px 4px;
+  color: rgba(255,255,255,.75);
```
Comment (jessicaslaughter) [inline portfolio/src/main/webapp/style.css]: nit: it looks like some indentation is off here

**Review by jessicaslaughter — CHANGES_REQUESTED** [review]

**Comment by jessicaslaughter** [comment]
could you also add some screenshots or a quick video demo of what your portfolio looks like at this PR to the PR description?

**Comment by gracetian6** [comment]
Here is what my website looks like
https://drive.google.com/file/d/1GbjcWOeASgguyWwTuSw4dYA66BEVfFEC/view?usp=sharing

File: portfolio/src/main/webapp/data.js [hunk 9]
```diff
@@ -1,66 +1,69 @@
 // arrays store text that will be displayed on pages 
 window.moments = [
-    `I did math research on knots in high school, 
-    so my goal was to prove new patterns about these 
-    geometrical figures. So, I spent a lot of time 
-    drawing a lot of diagrams to test out new ideas.`,
+    'I did math research on knots in high school,'
+    + 'so my goal was to prove new patterns about '
+    + 'these geometrical figures. So, I spent a lot '
+    + 'of time drawing a lot of diagrams to test out new ideas.',
```
Comment (jpnickolas) [inline portfolio/src/main/webapp/data.js:7]: Good use of whitespace.

File: portfolio/src/main/webapp/script.js [hunk 10]
```diff
@@ -66,7 +66,8 @@ function constructImage(imageIndex){
 
   // Add Caption
   const txtElement = document.createElement('p');
-  txtElement.innerText = window.moments[imageIndex];
+  console.log(window.moments[imageIndex]);
```
Comment (jpnickolas) [inline portfolio/src/main/webapp/script.js]: Remove debug statement.

**Review by jpnickolas — CHANGES_REQUESTED** [review]

**Comment by jpnickolas** [comment]
Overall looks good. You should also try deploying to your GCP project and see if you can send us a link to try out.

File: portfolio/src/main/webapp/script.js [hunk 11]
```diff
@@ -66,7 +66,7 @@ function constructImage(imageIndex){
 
   // Add Caption
   const txtElement = document.createElement('p');
-  txtElement.innerText = window.moments[imageIndex];
+  txtElement.innerText = window.moments[imageIndex];
```
Comment (jpnickolas) [inline portfolio/src/main/webapp/script.js:69]: Remove trailing whitespace

File: portfolio/pom.xml [hunk 12]
```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
-          <deploy.projectId>YOUR_PROJECT_ID_HERE</deploy.projectId>
+          <deploy.projectId>google.com:gtian-step-2020</deploy.projectId>
```
Comment (jpnickolas) [inline portfolio/pom.xml:38]: nit: use new project.

**Review by jpnickolas — APPROVED** [review]

**Pull request closed without merging**